lte/gateway/deploy/magma_dev_focal.yml

```diff
@@ -48,6 +48,5 @@
         magma_repo: /home/{{ ansible_user }}/magma-packages
         magma_deps: /home/{{ ansible_user }}/magma-deps
     - role: fluent_bit
-    - role: envoy
```

Comment:
If you remove it from there It won't be install in the devs VM.

Reply:
@tmdzk the magma role would install it from our repo.